apps/extension/content/ContentApp.tsx

```diff
@@ -139,8 +146,10 @@ export default function ContentApp({
 
 		return () => {
 			document.removeEventListener("mousemove", () => {});
+			document.removeEventListener('keydown', handleKeyDown, true);
+
 		};
-	}, []);
+	}, [isPopoverOpen]);
```

Comment:
Why should the code re run if the popover is opened or closed, I feel it's unnecessary

Reply:
I see, fair that we won't need to run the code again and again. Will push the commit